### PR TITLE
added network recovery mechanism

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

**Comment by MeirShpilraien** [comment]
- [ ] add tests
- [ ] add shard failure recovery

**Comment by MeirShpilraien** [comment]
@itamarhaber can you pls review this, I will leave the shard failure recovery to another PR.

File: src/cluster.c [hunk 1]
```diff
@@ -571,20 +674,47 @@ int Cluster_GetClusterInfo(RedisModuleCtx *ctx, RedisModuleString **argv, int ar
         }else{
             RedisModule_ReplyWithStringBuffer(ctx, "None", strlen("None"));
         }
+        RedisModule_ReplyWithStringBuffer(ctx, "runid", strlen("runid"));
```
Comment (itamarhaber) [inline src/cluster.c]: Good thing that RG.GETCLUSTERINFO is not documented or you'd have had to update the docs because of this change in the reply :)

Comment (MeirShpilraien) [reply src/cluster.c]: Prefer not to document it yet, it might change and it is not something that I want users to use.

**Comment by MeirShpilraien** [comment]
@itamarhaber fixed.

**Pull request merged**